testdata/picard/fingerprint/NA12878_chr1_fingerprint.vcf

```diff
@@ -0,0 +1,3399 @@
+##contig=<ID=HLA-DQA1*05:03,length=6121,assembly=38>
+##contig=<ID=HLA-DQA1*05:05:01:01,length=6593,assembly=38>
+##contig=<ID=HLA-DQA1*05:05:01:02,length=6597,assembly=38>
+##contig=<ID=HLA-DQA1*05:05:01:03,length=6393,assembly=38>
+##contig=<ID=HLA-DQA1*05:11,length=6589,assembly=38>
+##contig=<ID=HLA-DQA1*06:01:01,length=5878,assembly=38>
+##contig=<ID=HLA-DQB1*02:01:01,length=7480,assembly=38>
+##contig=<ID=HLA-DQB1*02:02:01,length=7471,assembly=38>
+##contig=<ID=HLA-DQB1*03:01:01:01,length=7231,assembly=38>
+##contig=<ID=HLA-DQB1*03:01:01:02,length=7230,assembly=38>
+##contig=<ID=HLA-DQB1*03:01:01:03,length=7231,assembly=38>
+##contig=<ID=HLA-DQB1*03:02:01,length=7126,assembly=38>
+##contig=<ID=HLA-DQB1*03:03:02:01,length=7126,assembly=38>
+##contig=<ID=HLA-DQB1*03:03:02:02,length=7126,assembly=38>
+##contig=<ID=HLA-DQB1*03:03:02:03,length=6800,assembly=38>
+##contig=<ID=HLA-DQB1*03:05:01,length=6934,assembly=38>
+##contig=<ID=HLA-DQB1*05:01:01:01,length=7090,assembly=38>
+##contig=<ID=HLA-DQB1*05:01:01:02,length=7090,assembly=38>
+##contig=<ID=HLA-DQB1*05:03:01:01,length=7089,assembly=38>
+##contig=<ID=HLA-DQB1*05:03:01:02,length=7089,assembly=38>
+##contig=<ID=HLA-DQB1*06:01:01,length=7111,assembly=38>
+##contig=<ID=HLA-DQB1*06:02:01,length=7102,assembly=38>
+##contig=<ID=HLA-DQB1*06:03:01,length=7103,assembly=38>
+##contig=<ID=HLA-DQB1*06:09:01,length=7102,assembly=38>
+##contig=<ID=HLA-DRB1*01:01:01,length=10741,assembly=38>
+##contig=<ID=HLA-DRB1*01:02:01,length=11229,assembly=38>
+##contig=<ID=HLA-DRB1*03:01:01:01,length=13908,assembly=38>
+##contig=<ID=HLA-DRB1*03:01:01:02,length=13426,assembly=38>
+##contig=<ID=HLA-DRB1*04:03:01,length=15246,assembly=38>
+##contig=<ID=HLA-DRB1*07:01:01:01,length=16110,assembly=38>
+##contig=<ID=HLA-DRB1*07:01:01:02,length=16120,assembly=38>
+##contig=<ID=HLA-DRB1*08:03:02,length=13562,assembly=38>
+##contig=<ID=HLA-DRB1*09:21,length=16039,assembly=38>
+##contig=<ID=HLA-DRB1*10:01:01,length=13501,assembly=38>
+##contig=<ID=HLA-DRB1*11:01:01,length=13921,assembly=38>
+##contig=<ID=HLA-DRB1*11:01:02,length=13931,assembly=38>
+##contig=<ID=HLA-DRB1*11:04:01,length=13919,assembly=38>
+##contig=<ID=HLA-DRB1*12:01:01,length=13404,assembly=38>
+##contig=<ID=HLA-DRB1*12:17,length=11260,assembly=38>
+##contig=<ID=HLA-DRB1*13:01:01,length=13935,assembly=38>
+##contig=<ID=HLA-DRB1*13:02:01,length=13941,assembly=38>
+##contig=<ID=HLA-DRB1*14:05:01,length=13933,assembly=38>
+##contig=<ID=HLA-DRB1*14:54:01,length=13936,assembly=38>
+##contig=<ID=HLA-DRB1*15:01:01:01,length=11080,assembly=38>
+##contig=<ID=HLA-DRB1*15:01:01:02,length=11571,assembly=38>
+##contig=<ID=HLA-DRB1*15:01:01:03,length=11056,assembly=38>
+##contig=<ID=HLA-DRB1*15:01:01:04,length=11056,assembly=38>
+##contig=<ID=HLA-DRB1*15:02:01,length=10313,assembly=38>
+##contig=<ID=HLA-DRB1*15:03:01:01,length=11567,assembly=38>
+##contig=<ID=HLA-DRB1*15:03:01:02,length=11569,assembly=38>
+##contig=<ID=HLA-DRB1*16:02:01,length=11005,assembly=38>
```

Comment:
Is it possible to remove header lines that are not specific to Chr1?

Reply:
good call, done